index.js

```diff
@@ -20,5 +20,15 @@ module.exports = async (config) => {
     if (config.help) return helpers.printHelpMessage();
     if (config.version) return helpers.printVersion();
 
-    return await helpers.analyze(config);
+    try {
+        const returnCode = await helpers.analyze(config);
+        if (returnCode === 0 || returnCode === undefined) {
```

Comment:
Thoughts on the pros and cons of `returnCode === undefined` being treated as no problems in contrast to there was some problem? 

If analyze somehow returns undefined, wouldn't we want to know that?

Reply:
Good question.
I wrote it for possible non-returns which can be caused by my mistake.

However, as you said, we should handle this with logging and abnormal termination.
I will do that, thx!